cl/phase1/forkchoice/mock_services/forkchoice_mock.go

```diff
@@ -1,20 +1,22 @@
-package forkchoice
+package mock_services
```

Comment:
@Giulio2002 @domiwei fyi I had to move `forkchoice_mock.go` into a sub-package because there was a bit of an annoying circular mock dependency with `go generate`